pandas/_libs/tslibs/offsets.pyx

```diff
@@ -49,7 +47,9 @@ from pandas._libs.tslibs.timezones cimport utc_pytz as UTC
 from pandas._libs.tslibs.tzconversion cimport tz_convert_single
 
 from .timedeltas cimport delta_to_nanoseconds
-
+from .timedeltas import Timedelta
```

Comment:
are we consisten about relative imports? I *really* don't like mixing this.

Reply:
we're not, but the above line is relative so i figured this should be too since its the same module.

while we generally do absolute, i think we should move to relative for tslibs since self-containment is a big deal here